angular-app/src/proxy.conf.js

```diff
@@ -0,0 +1,10 @@
+const PROXY_CONFIG = [
```

Comment:
This file is specifically for setting up your dev server, right? Does the file name have to be exactly what it currently is? Or could you maybe rename it to `dev_proxy.conf.js` for a little more clarity on what this is supposed to be used for?